nlp_modules/seq2set_entity_recognizer.py

```diff
@@ -0,0 +1,213 @@
+"""
+seq2set_entity_recognizer.py
+
+Resolves entity types from conllu parses.
+"""
+from nlp_modules.base import NLPModule, PipelineDep
+from ssn.args import eval_argparser
+from ssn import input_reader
+from ssn.config_reader import process_configs
+from seq2set_evaluator import SSNTrainer
+
+
+class Seq2setNNER(NLPModule):
+    """
+    Input: Conllu
+    Output: Conllu
+    """
+    requires = (PipelineDep.PARSE,)
+    provides = (PipelineDep.PARSE,)
```

Comment:
Is this right? If this module produces entity types given a parse, I think it should be providing something like `PipelineDep.ENTITY_TYPES`

Reply:
I think that's relevant to what is fed into the coreference system. If we plan to change to a neural coreference resolver, we may also need the conllu format as input.